src/app/clusters/ota-requestor/DefaultOTARequestorDriver.h

```diff
@@ -92,6 +93,7 @@ class DefaultOTARequestorDriver : public OTARequestorDriver
     void ScheduleDelayedAction(System::Clock::Seconds32 delay, System::TimerCompleteCallback action, void * aAppState);
     void CancelDelayedAction(System::TimerCompleteCallback action, void * aAppState);
     bool ProviderLocationsEqual(const ProviderLocationType & a, const ProviderLocationType & b);
+    CHIP_ERROR ScheduleRetry(bool trySameProvider);
```

Comment:
Probably could add a function header talking about return value of `CHIP_NO_ERROR` indicates a query retry has been successfully scheduled.

Reply:
Added function header comment.